app/src/main/res/values/strings.xml

```diff
@@ -440,6 +440,9 @@ Upload your first media by tapping on the add button.</string>
   <string name="ends_on">Ends on:</string>
   <string name="display_campaigns">Display campaigns</string>
   <string name="display_campaigns_explanation">See the ongoing campaigns</string>
+  <string name="get_content_photo_picker_title">Use GET_CONTENT photo picker</string>
+  <string name="get_content_photo_picker_explanation">Disable if your pictures get uploaded without location</string>
+  <string name="location_loss_warning">Please make sure that this new Android picker does not strip location from your pictures.</string>
```

Comment:
This message isn't going to help much as we can't expect the users to know what the "new Android picker" means and whether / not it strips location. 

It would be great to tweak this in a way that helps them in some way. Some set of actionable steps to help them verify this would be better.

The only way to not strip locations while using the new picker is to use the "Browse.." option in the new picker's overflow menu. That does not have any location stripping issues since it goes to the Documents UI based picker. We could convey this may be in the dialog.

Reply:
Personally I believe this is good enough for now.
Switching this UI element makes a popup with a help link appear. So people who want to know the details will be able to get the full details on that page. The setting's title and explanation should focus on the most essential.
I believe users will naturally understand that `this new Android picker` refers to the GET_CONTENT photo picker, as this message only appears when enabling it.

Reply:
It's fine if that we do not need to explain in much detail in the dialog given we have an accompanying help page. My concern is that , description and dialog text expect a bit more Android knowledge that seems ideal. This has been the base of my suggestions and I don't understand how they're not an improvement of the status quo.

> I believe users will naturally understand that this new Android picker refers to the GET_CONTENT photo picker, ...

My whole point is we can't expect users to understand what `GET_CONTENT ` even means unless someone explains it to them. To be clear, I'm not suggesting to explain here what GET_CONTENT picker means. I'm only suggesting to use phrasing that would reduce the need for us to even do that :-)

Reply:
> Please make sure that the current setting option does not strip location from your pictures

Additionally giving some hint as to how they could identify that locations are stripped (by checking the "Read More" link) would be helpful.